### PR TITLE
Web streaming example

### PR DESCRIPTION
**Pull request by raulraja**
This PR shows streaming works through the Xef server.

https://github.com/xebia-functional/xef/assets/456796/4c056187-b1d9-429c-9941-55c61cfb0f50

### CODE REVIEW

**Review by javipacheco — APPROVED** [review]
🚀 LGTM!

I have tried the code locally and everything works as expected

**Review by Montagon — DISMISSED** [review]
🚀

File: server/web/src/vite-env.d.ts [hunk 1]
```diff
@@ -1,296 +0,0 @@
-/// <reference types="vite/client" />
-
-type Settings = {
```
Comment (javipacheco) [inline server/web/src/vite-env.d.ts:3]: I'm not sure what is the problem, VSCode complains about not finding this type but I can launch the project fine. 

Should we keep this type in this file?

Comment (raulraja) [reply server/web/src/vite-env.d.ts:3]: I don't know 🤷

Comment (raulraja) [reply server/web/src/vite-env.d.ts:3]: Readded the Settings object

**Comment by raulraja** [comment]
Included additional chat example with no memory

**Review by javipacheco — APPROVED** [review]
🚀

**Pull request merged**